diff --git a/node_modules/trystero/src/strategy.js b/node_modules/trystero/src/strategy.js
index 965356e..418e252 100644
--- a/node_modules/trystero/src/strategy.js
+++ b/node_modules/trystero/src/strategy.js
@@ -14,8 +14,8 @@ import {
   watchOnline
 } from './utils.js'
 
-const poolSize = 20
-const announceIntervalMs = 5_333
+const poolSize = 2
+const announceIntervalMs = 8_000
 const offerTtl = 57_333
 
 export default ({init, subscribe, announce}) => {
